tests/cookbook/test_write.py

```diff
@@ -24,12 +28,92 @@ def test_parquet_write_with_partitioning(tmp_path):
     pd_df = df.write_parquet(tmp_path, partition_cols=["Borough"])
 
     read_back_pd_df = daft.read_parquet(tmp_path.as_posix() + "/**/*.parquet").to_pandas()
-    assert_df_equals(df.exclude("Borough").to_pandas(), read_back_pd_df)
```

Comment:
So the new writer doesn't prune partition columns from the data files?

Reply:
Yup to be consistent with Iceberg